packages/cli/src/ceramic-daemon.ts

```diff
@@ -323,12 +325,17 @@ export class CeramicDaemon {
    */
   async createStreamFromGenesis(req: Request, res: Response): Promise<void> {
     const { type, genesis, opts } = req.body
-    const stream = await this.ceramic.createStreamFromGenesis(
-      type,
-      StreamUtils.deserializeCommit(genesis),
-      opts
-    )
-    res.json({ streamId: stream.id.toString(), state: StreamUtils.serializeState(stream.state) })
+    try {
+      const stream = await this.ceramic.createStreamFromGenesis(
+        type,
+        StreamUtils.deserializeCommit(genesis),
+        opts
+      )
+      res.json({ streamId: stream.id.toString(), state: StreamUtils.serializeState(stream.state) })
+    } catch (e) {
+      console.error(e)
```

Comment:
This feels like a weird change to come in this PR, and only to this one function but not all the other similar functions in this file.  Seems like we should have a separate PR that adds error logging to all HTTP endpoints if it's not already there?  Although TBH I thought we already did log errors as part of the http handler

Reply:
"fog of war" remnant. restored.